### DISCUSSION TITLE
Error 500 when making a root request to the webservice in JSON format

### DISCUSSION THREAD

**Discussion by callart123** [PrestaShop/PrestaShop#33121] [Q&A]
Hello,

Since the Prestashop 8 update, I've had a 500 error on one when I make a request to the prestashop api. 

Previously my request was 
"shopurl/api/?output_format_format=JSON" 
worked and returned a status of 200, now I get a status of 500. 

I don't get this error if I send my request to a resource, for example "shopurl/api/orders/?output_format_format=JSON" works.

Do you have any idea how to get around this?

Thanks in advance for your suggestions!

**Comment by kpodemski** [comment]
Hello @callart123 

Could you enable developer mode and try again? You should be able to see a more informative error thanks to that.

**Reply by korberg** [reply]
had similar issue - accessing and updating products via webservice. the error disappeared when i switched debug mode on. seems to have gone when now switched off... 
the error did not appear at all in my dev-server, where dev mode is alway on.
thnx!
